ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml

```diff
@@ -30,4 +30,12 @@ spec:
           exit 1
         fi
 
+        ISV_PID=$(cat $CI_FILE_PATH | yq -r '.isv_pid')
```

Comment:
We should get the `isv_pid` from their certProject. We already have the project id to do that lookup.

Reply:
Okay, so make another request in the reserve-operator-name script to GET the certProject data?

Reply:
A better thing would be to retrieve the certProject record, even if just via curl, in this step and store it in a workspace. Then it can be referenced again in followup steps and tasks.